src/main/java/algorithms/minimumSpanningTree/prims/Edge.java

```diff
@@ -0,0 +1,28 @@
+package algorithms.minimumSpanningTree.prims;
+
+/**
+ * Helper class to encapsulate information about an edge;
+ * specifically, tells us the weight of an edge and the node on the other end
+ * (assumes that the starting node of the edge is known)
+ */
+public class Edge {
```

Comment:
oh? wasn't our discussion to do away with the edge class and simply have a Node class with a graph representation using adjacency list / matrix? This way we can circumvent the issue of having directed / undirected graphs while not being too verbose.

But if you think an keeping an Edge class would be good (i guess, for one, it aligns with prof seth's notes), i suggest an edge should capture Node1 and Node2 (direction can be defined as Node1 -> Node2). My only concern is while this makes alot of encapsulation sense, it might make the code too verbose and unnec. long

What do you think?